2017/ru/0x00-header.md

```diff
@@ -0,0 +1,19 @@
+![OWASP LOGO](images/OWASP_logo.png)
+
+## Топ-10 OWASP 2017
+
+Десять самых опасных угроз безопасности веб-приложений
+
+
+
+
```

Comment:
Please remove useless empty breaks. It should exactly one empty string here.

Reply:
OK. Done